Localization/newrun.py

```diff
@@ -0,0 +1,313 @@
+# -*- coding: utf-8 -*-
```

Comment:
How much changed in this file with respect to the old move.py file? If not much changed, please overwrite move.py to avoid duplicates.